lib/components/atoms/Track/types/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
```

Comment:
Why only this Prop? Please define your props here that components receives

Reply:
done in separate